native-file-windows.txt

```diff
@@ -11,4 +11,4 @@ endian = 'little'
 [properties]
 c_args = ['-D_WIN32_WINNT=0x0601', '-march=x86-64']
 cpp_args = ['-D_WIN32_WINNT=0x0601', '-march=x86-64']
-nuget-download = false
+nuget-download = true 
```

Comment:
wow, do you know we it was set to `false`?

Reply:
yeah, it was `false` because it wasn't really needed until now. Emile requires jpeg dependency.